app/assets/javascripts/channels/main_page.js

```diff
@@ -0,0 +1,36 @@
+$(function () {
+
+  'use strict';
+
+  var organization = $('#current_organization').text().trim();
+
+  if (location.pathname === "/" && location.search === '') {
+    var current_user_id = $('.current_user_id').data('id');
+    App.cable.subscriptions.create({
+      channel: "MainPageChannel",
+      organization: organization
+    }, {
+      received: function(data) {
+        if (!(data['user_id'] === current_user_id)) {
+          AddTransactionToList (
+            data['id'],
+            data['view'],
+            data['sidebar'],
+            data['total_balance']
+          );
+        }
+      }
+    });
+  }
+
+  function AddTransactionToList (element_id, element, sidebar, total_balance) {
```

Comment:
@dissident move this to `transactions.js`. Try to refactor `transactions/create.js.erb` to use this method too. (if it will not take too many time).